Manifest.toml

```diff
@@ -29,9 +29,9 @@ version = "0.2.0"
 
 [[CompilerSupportLibraries_jll]]
 deps = ["Libdl", "Pkg"]
-git-tree-sha1 = "b57c5d019367c90f234a7bc7e24ff0a84971da5d"
+git-tree-sha1 = "aa832564f930a7fc9290972526908d01a35aefac"
```

Comment:
Not sure what's going on here. Should I expect this to have changed?

Reply:
Not unless you did some package operation like `up` or `add`. You could probably just discard these changes (but I doubt it's a problem either way).